File: aurora-scala/src/main/scala/net/gree/aurora/scala/domain/cluster/TaggedClusterIdResolverRepository.scala
package net.gree.aurora.scala.domain.cluster

import org.sisioh.dddbase.core.lifecycle.sync.SyncRepository

trait TaggedClusterIdResolverRepository[HINT]
  extends SyncRepository[TaggedClusterIdResolverId, TaggedClusterIdResolver[HINT]] {

}

File: aurora-core/src/main/java/net/gree/aurora/domain/cluster/TaggedClusterIdResolverRepository.java
package net.gree.aurora.domain.cluster;

import org.sisioh.dddbase.lifecycle.sync.SyncEntityIterableReader;
import org.sisioh.dddbase.lifecycle.sync.SyncRepository;

public interface TaggedClusterIdResolverRepository<HINT>
        extends SyncRepository<TaggedClusterIdResolverRepository<HINT>, TaggedClusterIdResolverId, TaggedClusterIdResolver<HINT>>,
        SyncEntityIterableReader<TaggedClusterIdResolverId, TaggedClusterIdResolver<HINT>> {
}
